src/IB/IBFESurfaceMethod.cpp

```diff
@@ -430,28 +430,44 @@ IBFESurfaceMethod::preprocessIntegrateData(double current_time, double new_time,
 void
 IBFESurfaceMethod::postprocessIntegrateData(double /*current_time*/, double /*new_time*/, int /*num_cycles*/)
 {
-    batch_vec_ghost_update(
-        {
-            d_X->new_vecs, d_U->new_vecs, d_U_n->new_vecs, d_U_t->new_vecs, d_F->half_vecs
-#if (d_use_pressure_jump_conditions)
-                ,
-                d_P_jump->half_vecs, d_P_in->half_vecs, d_P_out->half_vecs
-#endif
-#if (d_use_velocity_jump_conditions)
-                ,
-                d_WSS_in->half_vecs, d_WSS_out->half_vecs, d_DU_jump[0]->half_vecs, d_DU_jump[1]->half_vecs
+    //~ batch_vec_ghost_update(
```

Comment:
Just get rid of this --- don't comment it out.

Reply:
aah i wiped it out but apparently I committed before saving!